.github/workflows/nix.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       with:
         name: enarx
         authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-    - run: nix run -L . info
+    - run: nix run -L . cpu info
```

Comment:
Does this belong in this PR/commit? What's the reasoning here?

Reply:
I saw that this test was failing due to the lack of this change, and rather than file a whole new PR for this I just stuffed it in here. Do you want me to extract it into its own PR?

Reply:
I've made it into its own commit.